### PR TITLE
Feature/organize leads from login to home screen

### PR DESCRIPTION
**Pull request by Hiroki0613**
This is PR from Issue.
↓↓↓
https://github.com/KingdomSeekersJP/salon_app/issues/10

This is the movie.
https://youtu.be/ZvhXhlrpMKM

### CODE REVIEW

File: creator/lib/screens/salon_application_screen.dart [hunk 1]
```diff
@@ -11,24 +13,35 @@ class SalonApplicationScreen extends StatefulWidget {
...
+        _noFieldsEmpty = (firstNameController.text.isNotEmpty &&
+            lastNameController.text.isNotEmpty &&
+            emailController.text.isNotEmpty &&
+            reasonController.text.isNotEmpty);
+      },
+    );
   }
 
   @override
   Widget build(BuildContext context) {
```
Comment (dragonfly91) [inline creator/lib/screens/salon_application_screen.dart:37]: method too long. better to create a separate method to build the body

**Review by dragonfly91 — APPROVED** [review]
Left one minor comment. Otherwise LGTM.

File: creator/lib/screens/salon_application_screen.dart (hunk 1 above)
Comment (Hiroki0613) [reply creator/lib/screens/salon_application_screen.dart:37]: I separate the method.

**Review by dragonfly91 — APPROVED** [review]
LGTM

**Pull request merged**